src/EventHandler/Story/StoryReaction.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of MadelineProto.
+ * MadelineProto is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ * MadelineProto is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with MadelineProto.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author    Amir Hossein Jafari <amirhosseinjafari8228@gmail.com>
+ * @copyright 2016-2023 Amir Hossein Jafari <amirhosseinjafari8228@gmail.com>
+ * @license   https://opensource.org/licenses/AGPL-3.0 AGPLv3
+ * @link https://docs.madelineproto.xyz MadelineProto documentation
+ */
+
+namespace danog\MadelineProto\EventHandler\Story;
+
+use danog\MadelineProto\EventHandler\AbstractStory;
+use danog\MadelineProto\MTProto;
+
+/**
+ *
+ */
+final class StoryReaction extends AbstractStory
```

Comment:
I think this should not extend `AbstractStory`, since it does not represent a story.

Reply:
I did it because it has same user id & story id.

if you can it is not important I will fix it

Reply:
?

Reply:
> ?

I told you I think it is better to be an AbstractStory for all of them.
specially when there is same parameters

Reply:
AbstractStory should represent different kinds of stories, a story reaction is not a story, even if it refers to a story.